developer_guide/installation/system_requirements/setup_requirements_ubuntu_1404.rst

```diff
@@ -0,0 +1,117 @@
+Setup System Requirements on Ubuntu 14.04
+==========================================
+
+Here is a quick guide to setup the :doc:`system_requirements` requirements on Ubuntu 14.04.
+
+System installation
+-------------------
+
+Base dependencies
+*****************
+
+Before installing requirements, update repositories information. You'll have to install MySQL, Apache and PHP.
```

Comment:
Make sur you update your repositories information before installing any requirements.